src/Compilers/VisualBasic/Test/Semantic/Semantics/IteratorTests.vb

```diff
@@ -2,14 +2,61 @@
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
 
+Imports Microsoft.CodeAnalysis.Test.Utilities
 Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
 Imports Roslyn.Test.Utilities
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
 
     Public Class IteratorTests
         Inherits FlowTestBase
 
+        <Fact>
+        Public Sub BasicIterator()
+            Dim compilation = CreateCompilation(
+                    <compilation>
+                        <file name="a.vb">
+                            <![CDATA[
+Imports System.Collections.Generic
+
+Class C
+    Public Iterator Function I As IEnumerable(Of Integer)
```

Comment:
>Public Iterator Function I As IEnumerable(Of Integer) [](http://example.com/codeflow?start=4&length=53)

VB also supports iterator lambdas and properties, I think we need to cover those too.